kdwsdl2cpp/src/compiler.cpp

```diff
@@ -103,7 +103,6 @@ void Compiler::parse(const QDomElement &element)
 
         definitions.fixUpDefinitions(/*&context, element*/);
 
-        KODE::Code::setDefaultIndentation(4);
```

Comment:
Why was this removed?

Reply:
Not removed, just moved into the Creator class.